src/components/main/main.e2e.test.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Enzyme, {mount} from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import Main from './main.jsx';
+
+Enzyme.configure({
+  adapter: new Adapter(),
+});
+
+const mock = {
+  promoTitle: `Breaking Bad`,
+  promoGenre: `drama`,
+  promoReleaseDate: 2008,
+  movieTitlesArr: [`We should talk about kevin`, `HuperuCube`, `Jill`]
+};
+
+const onTitleClick = jest.fn();
+
+describe(`Main Component E2E`, () => {
+  it(`correct amount of cards should be rendered`, () => {
+    const main = mount(
+        <Main
+          {...mock}
+          onMovieTitleClick={onTitleClick}
+        />
+    );
+
+    const movieCards = main.find(`.small-movie-card`);
+    expect(movieCards.length).toBe(mock.movieTitlesArr.length);
+  });
+
+  it(`title must be pressed`, () => {
+    const main = mount(
+        <Main
+          {...mock}
+          onMovieTitleClick={onTitleClick}
+        />
+    );
+
+    main.find(`.small-movie-card__title`).forEach((it) => it.simulate(`click`));
+    expect(onTitleClick.mock.calls.length).toBe(mock.movieTitlesArr.length);
```

Comment:
Подскажи пож почему запись следующего вида не проходит тесты и ведет себя непредсказуемо? По виду должна отрабатывать корректно. Нагуглить не удалось

```suggestion
    main.find(`.small-movie-card__title`).forEach((it) => {
      it.simulate(`click`);
      expect(onTitleClick).toHaveBeenCalledTimes(1);
    };
```
![image](https://user-images.githubusercontent.com/30810655/85229414-27d5c500-b3f2-11ea-9f29-b63d280eff72.png) ![image](https://user-images.githubusercontent.com/30810655/85229434-420fa300-b3f2-11ea-8b3c-9704a025790c.png)

Reply:
вижу с этой  проблемойц ты справился.